.github/ISSUE_TEMPLATE/deprecation-tracker.yml

```diff
@@ -0,0 +1,65 @@
+name: ⚠️ Deprecation Tracker
+description: Use this template to flag and track deprecated items (field, model, or functionality) scheduled for removal.
+title: "[Deprecation] <Item Name>"
+labels: ["type:deprecation"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for helping manage our deprecation process. This template is intended to track deprecated items and plan for their ultimate removal. Providing complete information below will help us ensure a smooth transition and keep our users informed.
+  - type: input
+    attributes:
+      label: Planned Removal Date
+      description: Enter the date when this deprecated item will be fully removed (format: Month Day, Year. eg. May 10th, 2025).
```

Comment:
Have we ever seen this error before? Seems to be an issue with the parentheses. 

<img width="1434" alt="Screenshot 2025-03-04 at 4 23 51 PM" src="https://github.com/user-attachments/assets/80757609-0050-4426-b038-303816e2e9c2" />

Reply:
Good catch, it looks like it didn't like the `:` on line 13. Removed the colon and rephrased that bit to be a little more clear regarding the format expected/requested.